packages/services/src/processDPP.service.ts

```diff
@@ -60,7 +62,7 @@ export const processDPP: IService = async (data: any, context: IDppContext): Pro
       deleteItemFromLocalStorage(context.localStorageParams);
     }
 
-    return { vc, linkResolver };
+    return { vc, decodedEnvelopedVC, linkResolver };
```

Comment:
Should we check the `linkResolver`? Because I can not verify it on the verify page
<img width="1422" alt="image" src="https://github.com/user-attachments/assets/bdddfc1a-257e-44e2-a5a3-c0eda2da159a">

This is the error
<img width="1440" alt="image" src="https://github.com/user-attachments/assets/3c72af12-c3d1-490d-9959-3ba76512f44e">

Reply:
Hi @ldhyen99, I'm not sure I understand your comment. You are asking if we should validate the `linkResolver` object, but in your photos, it looks like the credential is not being registered with the storage service. Could you elaborate?

Reply:
I believe @ldhyen99 tested several times with this identity(the GTIN), so the responded URL was an old and removed stored DPP link. I think you need to clean your local services

Reply:
When I click on the `http://localhost:3005/gs1/gtin/05012345678900/10/2024001/TRF240001?linkType=gs1:certification` on localstorage, it redirects to verify page. In the `verify page` page, I see the error as in the image above, maybe this link was not stored in Storage Service.